llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1134,6 +1134,21 @@ static int getInstructionIDWithAttrMask(uint16_t *instructionID,
   return 0;
 }
 
+static bool isNFnotMap4(InternalInstruction *insn) {
+  // Promoted BMI instrs below has nf version.
+  if (insn->opcodeType == THREEBYTE_38 &&
+      ppFromEVEX3of4(insn->vectorExtensionPrefix[2]) == VEX_PREFIX_NONE) {
```

Comment:
Should we check if it's EVEX encoding?

Reply:
EVEX has been checked at line 1170.

Reply:
Missing check for `nfFromEVEX4of4(insn->vectorExtensionPrefix[3])`